src/terminal/docs/09-cheaper-by-the-dozen.md

```diff
@@ -403,7 +403,7 @@ sending it to `ls`.  So after expansion, the command would look like:
 
 If we want `ls` to find an asterisk something different is in order.
 
-The "\", or backslash, tells the shell to treat the following
```

Comment:
@mickfuzz it appears sometimes they are needed... :sweat_smile: I'm very happy about this, but it'll take me a while to review hehe :purple_heart: so please be patient